src/app/complex/complex-results/complex-list/complex-list.component.html

```diff
@@ -1,3 +1,34 @@
+<div class="Complex-navigator">
```

Comment:
In the ts file of this component, we need a new input `@Input() componentIds: Set<string>;`.

Then we can do here `*ngFor="let component of componentIds`, as we need the variable in the same component as where we use it.

Then in the HTML of ComplexResultsComponent, where we use `cp-complex-list`, we need to pass the list of component ids for the new input.
```
      <cp-complex-list [complexSearch]="complexSearch"
                       [componentIds]="allComponentsInComplexSearch"></cp-complex-list>
```